sdk/AzSdk.test.reference.props

```diff
@@ -0,0 +1,15 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why is this needed?

Reply:
The test Projects import it

Reply:
This should still be coming from the management toolset and not be added here.